EIPS/eip-7441.md

```diff
@@ -0,0 +1,101 @@
+---
+eip: 7441
+title: Upgrade block proposer election to Whisk
+description: Allow elected block proposers to remain private until block publishing, to prevent DoS attacks
+author: George Kadianakis (@asn-d6), Justin Drake (@JustinDrake), dapplion (@dapplion)
+discussions-to: https://ethereum-magicians.org/t/eip-7441-upgrade-block-proposer-election-to-whisk-ssle/15316
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-09-01
+---
+
+## Abstract
+
+Upgrades the block proposer election mechanism to Whisk, a single secret leader election (SSLE) protocol. Currently, block proposers are publicly known in advance, sufficiently to allow sequential DoS attacks that could disable Ethereum. This upgrade makes the next block proposer unknown until its block is published.
+
+## Motivation
+
+The beacon chain currently elects the next 32 block proposers at the beginning of each epoch. The results of this election are public and everyone gets to learn the identity of those future block proposers.
+
+This information leak enables attackers to launch DoS attacks against each proposer sequentially in an attempt to disable Ethereum.
+
+## Specification
+
```

Comment:
a `Parameters` section can be added to refer to constants relevant to describe the EIP for e.g. `WHISK_EPOCHS_PER_SHUFFLING_PHASE`

Reply:
a `<!-- TODO: add parameters. -->` kind of a TODO can be added which the bots can track post drafts so that the draft acceptance can become less burdensome

Reply:
If we can avoid having to list of constants that would be better, it would duplicate a lot of content from the specs

Reply:
i guess it makes sense , resolving this comment